src/components/biggive-campaign-card-filter-grid/biggive-campaign-card-filter-grid.tsx

```diff
@@ -356,7 +356,7 @@ export class BiggiveCampaignCardFilterGrid {
                     prompt="Beneficiary"
                     placeholder={this.beneficiariesPlaceHolderText}
                     selectedLabel={this.selectedFilterBeneficiary}
-                    options={this.beneficiaryOptions}
+                    options={this.optionsToArray(this.beneficiaryOptions || [])}
```

Comment:
The  `|| []` is probably not strictly needed, its in here really just because the test cases don't always pass all the props and adding this was easier than fixing the tests.